routes/hydrology.py

```diff
@@ -0,0 +1,307 @@
+            point_pkg[model_name][scenario_name] = dict()
+            for month_coord in dim_encodings["months"].keys():
+                month_name = dim_encodings["months"][month_coord]
+                point_pkg[model_name][scenario_name][month_name] = dict()
+                for era_coord in dim_encodings["eras"].keys():
+                    era_name = dim_encodings["eras"][era_coord]
+                    point_pkg[model_name][scenario_name][month_name][era_name] = dict()
+                    for var_coord in dim_encodings["varnames"].keys():
+                        var_name = dim_encodings["varnames"][var_coord]
+                        point_pkg[model_name][scenario_name][month_name][era_name][
+                            var_name
+                        ] = rasdaman_response[model_coord][scenario_coord][month_coord][
+                            era_coord
+                        ].split(
+                            " "
+                        )[
+                            var_coord
+                        ]
+
+    return point_pkg
+
+
+def run_fetch_hydrology_point_data_mmm(lat, lon):
+    """Fetch hydrology data for a
+       given latitude and longitude, and summarize over eras.
+
+    Args:
+        lat (float): latitude
+        lon (float): longitude
+
+    Returns:
+        JSON-like dict of data at provided latitude and
+        longitude for all variables
+    """
+    # get standard point data package
+    point_pkg = run_fetch_hydrology_point_data(lat, lon)
+
+    # compute min/mean/max across eras for each variable name
+    mmm_dict = dict()
+
+    var_names = []
+    for var_coord in dim_encodings["varnames"].keys():
+        var_names.append(dim_encodings["varnames"][var_coord])
+
+        for var_name in var_names:
+            values = []
+
+            for model_coord in dim_encodings["models"].keys():
+                model_name = dim_encodings["models"][model_coord]
+                for scenario_coord in dim_encodings["scenarios"].keys():
+                    scenario_name = dim_encodings["scenarios"][scenario_coord]
+                    for month_coord in dim_encodings["months"].keys():
+                        month_name = dim_encodings["months"][month_coord]
+                        for era_coord in dim_encodings["eras"].keys():
+                            era_name = dim_encodings["eras"][era_coord]
+
+                            values.append(
+                                float(
+                                    point_pkg[model_name][scenario_name][month_name][
+                                        era_name
+                                    ][var_name]
+                                )
+                            )
+
+            min_value, mean_value, max_value = (
+                min(values),
+                round(np.nanmean(values), 2),
+                max(values),
+            )
+
+            mmm_dict[var_name] = min_value, mean_value, max_value
```

Comment:
This block might need to be indented another level? I'm currently fetching duplicate values across scenarios for a `mmm` query like this:

http://localhost:5000/hydrology/point/61.20663/-148.95676?summarize=mmm

![image](https://github.com/ua-snap/data-api/assets/7774271/5b82d607-1455-413a-8b7a-ec5cd88924be)

I guess this could be happening in the packaging component too, but that part looked straightforward on my first reading.

Reply:
Oh on closer inspection these values are duped between models as well:
![image](https://github.com/ua-snap/data-api/assets/7774271/f47c0aa0-ee9e-48fa-becb-cb6c8e6d77d6)

Reply:
I found the problem and was able to incorporate the mmm data grab inside of the repackaging instead.... fixed the problem with https://github.com/ua-snap/data-api/pull/358/commits/82e4b3ba34b49aeb88bc009af7332e32065e82a2 by _removing_ code for a change :) 
![image](https://github.com/ua-snap/data-api/assets/99696041/d68133fe-7755-427d-ba84-32a4130e331f)